openai/components/oai_finetune_pipeline_component.yml

```diff
@@ -1,5 +1,7 @@
 $schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
-name: microsoft_com_azureml_oaie_finetune_pipeline_v2
+
+name: microsoft_com_azureml_oai_finetune_pipeline_v2
```

Comment:
@elesel can you please check component naming as well? Should *microsoft_com_azureml* be omitted?

Reply:
![image](https://user-images.githubusercontent.com/61145377/234347972-db4fbd18-9ec8-480a-b51e-ac2c9eab353c.png)

https://github.com/Azure/azureml-assets/wiki/Asset-naming-convention

Reply:
@vikasagrawal-ms please check if component name can follow above standards. Thanks

Reply:
@novaturient95 , The name is ok, as reviewed by Naveen.

Reply:
@elesel can you please confirm

Reply:
@vikasagrawal-ms, please remove the `microsoft_com_azureml_` prefix. What @novaturient95 shared is correct.

Reply:
Also, versioning (v2) should be in the version field, not the environment name, unless there's a compelling reason otherwise.

Reply:
@elesel v2 is to suggest its DPv2 supported component

Reply:
All registry components are DPv2. Probably not worth including v2 in the name.

Reply:
Updated the name with openai_completions_pipeline after deliberation with PMs